src/mainwindow.cpp

```diff
@@ -3215,6 +3215,22 @@ void MainWindow::leaveEvent(QEvent *)
     this->unsetCursor();
 }
 
+/*!
+ * \brief MainWindow::changeEvent
+ */
+void MainWindow::changeEvent(QEvent *event)
+{
```

Comment:
When overriding virtual functions, we should generally not assume that the superclass doesn't need their implementation to be called, so I think we should add `MainWindowBase::changeEvent(event);`.

Reply:
Right, I always forget about that. 😓